### ISSUE TITLE
Clarification regarding Training Data Split and Fair Comparison in Table 1

### ISSUE THREAD

**Issue by yuni45** [valeoai/DrivoR#4]
I have a question regarding the experimental setup presented in the paper.

I noticed that all the methods compared in Table 1 (except from AutoVLA, DriveVLA-W0, and Ipad) strictly follow the official Navsim benchmark setting, utilizing only the 85k Navtrain samples for training. However, it appears that your method incorporates the 1.8k validation set into the training process.

Given that the official guideline specifies using the 85k training split, and including validation data often leads to significant performance gains, this raises a concern regarding the fairness of the comparison.

Could you please clarify the rationale behind merging the validation set into training? Furthermore, to ensure a fair comparison with the baselines, could you provide the performance metrics of your proposed method when trained solely on the 85k Navtrain dataset (without the validation set)?

Looking forward to your response. Thanks!

**Comment by EllingtonKirby** [comment]
Hi! Thanks for your interest in the paper, totally understand wanting to maintain integrity of the comparisons.

For reference the guidelines are quite clear from my point of view on using the `navtrain` split:

[NAVSIM v1](https://github.com/autonomousvision/navsim/blob/main/docs/submission.md)
[NAVSIM v2](https://huggingface.co/spaces/AGC2025/e2e-driving-warmup-iccv)

And quoting the challenge explicitly:
> For participation in the challenge, it is a strict requirement to register for your team by filling in this Google Form.
The leaderboard is now reopened.
You are limited to one upload per day on the challenge leaderboard, which should take approximately 2 hours to evaluate after a successful submission.
The deadline for submission is 23:59, UTC+0, on Sept. 20, 2025.
⚠️ IMPORTANT: Using the test/navtest/navhard_two_stage/warmup_two_stage/private_test_two_stage splits for training your challenge submissions is not allowed.
Using any other publicly available datasets or pretrained weights is allowed.
Furthermore, to be eligible for awards, the use of data must be described explicitly in the technical report for your submission.

And quoting directly from the NAVSIM [paper](https://arxiv.org/abs/2406.15349):
> We employ this filtering strategy to provide standardized splits for training and testing, called navtrain and navtest, with 103k
and 12k samples respectively. This curated data serves as a benchmark accessible as a standalone
download option with a moderate storage demand given its large scale and diversity (450 GB). 

Further, I am under the impression that most other works such as [Centaur](https://arxiv.org/abs/2503.11650), [DriveSuprim](https://arxiv.org/abs/2506.06659), [Hydra-MDP++](https://arxiv.org/abs/2503.12820), GTRS, ZTRS, etc. and many others use the 1192 logs of `navtrain`.

To be clear, at no point have we included any data outside of the 103k frames (1192 logs respectively) of the `navtrain` data split, which would really violate the spirit of the challenge.

We followed standard Machine Learning practices of validating our hyperparameters and model architecture using the `navtrain` validation set to conduct ablation studies. Once our final model architecture was decided, we trained on all available data (the entirety of `navtrain`) and we evaluate on the common benchmark `navtest`. 

However, we have launched a training which excludes the validation set from the `navtrain` data, and we will happily report the scores when it is complete.

**Comment by EllingtonKirby** [comment]
Hi there,

We have finished the training on only the 85k frames you specified. We stopped after 24 epochs, probably could get them higher by training longer.

```
All scores:
no_at_fault_collisions 0.989256
drivable_area_compliance 0.983451
ego_progress 0.899168
time_to_collision_within_bound 0.962457
comfort 1.0
driving_direction_compliance 0.970114
score 0.931288
```

<img width="858" height="102" alt="Image" src="https://github.com/user-attachments/assets/e005a540-c9f5-4ecc-b5b7-613dad437a82" />

<img width="1753" height="508" alt="Image" src="https://github.com/user-attachments/assets/0537f233-4126-40eb-b1bc-12c3b59d76a0" />


We provide the checkpoints [here ](https://github.com/valeoai/DrivoR/releases/download/model_weights/drivor_Nav1_train_only_85k_25epochs.ckpt)

Just to reiterate, the evaluation that is of concern for the challenge is the `navtest` score. I personally don't feel there is a reason to hold out the validation set once your model architecture and hyperparameters have been validated.
